consensus/hotstuff_leader.go

```diff
@@ -31,21 +31,28 @@ func (handler *HotstuffLeaderMessageHandler) HandleNewRoundMessage(m *consensusM
 	handler.emitTelemetryEvent(m, msg)
 
 	if err := handler.anteHandle(m, msg); err != nil {
-		m.nodeLogError(typesCons.ErrHotstuffValidation.Error(), err)
+		m.logger.Error().Msg(typesCons.ErrHotstuffValidation.Error())
 		return
 	}
 
 	// DISCUSS: Do we need to pause for `MinBlockFreqMSec` here to let more transactions or should we stick with optimistic responsiveness?
 
 	if err := m.didReceiveEnoughMessageForStep(NewRound); err != nil {
-		m.nodeLog(typesCons.OptimisticVoteCountWaiting(NewRound, err.Error()))
+		m.logger.Info().Msg(typesCons.OptimisticVoteCountWaiting(NewRound, err.Error()))
 		return
 	}
-	m.nodeLog(typesCons.OptimisticVoteCountPassed(m.height, NewRound, m.round))
+
+	m.logger.Info().Fields(
+		map[string]any{
+			"height": m.height,
+			"round":  m.round,
+			"step":   Prepare,
```

Comment:
Ditto https://github.com/pokt-network/pocket/pull/420#discussion_r1090059855

Reply:
s/Prepare/NewRound

This is inside the `HandleNewRoundMessage` function, so I think we should be logging `"step": NewRound` instead of `"step": Prepare` which we do  in the `HandlePrepareFunction`